### ISSUE TITLE
Test-extended.system-JDK11-aix_ppc-64_cmprssptrs SharedClassesAPI_0

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#3265]
https://ci.eclipse.org/openj9/job/Test-extended.system-JDK11-aix_ppc-64_cmprssptrs/40/tapResults/
```
SCC ************
SCC VERIFICATION FAILED
SCC ************
SCC ERROR: found 5 caches. Expected 4 caches
```

**Comment by hangshao0** [comment]
This is a test issue. The code is in openj9-systemtest repo. I opened an issue there: https://github.com/eclipse/openj9-systemtest/issues/46

**Comment by pshipton** [comment]
Resolved by eclipse/openj9-systemtest#48
